src/pal/inc/rt/palrt.h

```diff
@@ -1705,6 +1705,8 @@ typedef LONG (WINAPI *PTOP_LEVEL_EXCEPTION_FILTER)(
     );
 typedef PTOP_LEVEL_EXCEPTION_FILTER LPTOP_LEVEL_EXCEPTION_FILTER;
 
+BOOL VirtualUnwind(CONTEXT *context, KNONVOLATILE_CONTEXT_POINTERS *contextPointers);
```

Comment:
Maybe call it PAL_VirtualUnwind to make it clear that it is not a regular Win32 API?